eng/Subsets.props

```diff
@@ -63,6 +63,8 @@
     <!-- Even on platforms that do not support the CoreCLR runtime, we still want to build ilasm/ildasm. -->
     <DefaultCoreClrSubsets Condition="'$(PrimaryRuntimeFlavor)' != 'CoreCLR'">clr.iltools+clr.packages</DefaultCoreClrSubsets>
 
+    <DefaultNativeAotSubsets>clr.alljits+clr.tools+clr.nativeaotlibs+clr.nativeaotruntime</DefaultNativeAotSubsets>
```

Comment:
I think `UseNativeAotCoreLib` above might need a tweak. It controls what CoreLib should the libraries partition use for ApiCompat validation and similar.